config/300-pullsubscription.yaml

```diff
@@ -80,13 +80,13 @@ spec:
               type: string
               description: "ID of the Cloud Pub/Sub Topic to Subscribe to. It must be in the form of the unique identifier within the project, not the entire name. E.g. it must be 'laconia', not 'projects/my-gcp-project/topics/laconia'."
             ackDeadline:
-              type: string
+              type: integer
```

Comment:
this seems wrong? see the comments about units in the description?

Reply:
you are correct. It was half wired wrong... fixing.